### PR TITLE
[FEATURE] Put how_to_connect_to_sql_data fluent docs under test

### PR DESCRIPTION
**Pull request by TrangPham**
- [ ] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [x] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [x] Code is linted

    ```
    black .

    ruff . --fix
    ```
- [x] Appropriate tests and docs have been updated

For more details, see our [Contribution Checklist](https://docs.greatexpectations.io/docs/contributing/contributing_checklist), [Coding style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style), and [Documentation style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/docs_style).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

File: docs/docusaurus/docs/guides/connecting_to_your_data/fluent/database/how_to_connect_to_sql_data.md [hunk 1]
```diff
@@ -76,8 +74,8 @@ If you include a password as plain text in your connection string when you defin
 
 Creating a SQL Datasource is as simple as providing the `add_sql(...)` method a `name` by which to reference it in the future and the `connection_string` with which to access it.
 
-```python title="Python code"
-datasource = context.sources.add_sql(name="my_datasource", connection_string=sql_connection_string)
+
```
Comment (alexsherstinsky) [inline docs/docusaurus/docs/guides/connecting_to_your_data/fluent/database/how_to_connect_to_sql_data.md:77]: @TrangPham There is a slight inconsistency between the snippet displayed in the document dynamically and another example appearing in the document (statically) -- in terms of the database driver.  Please open the Netlify view in order to see this.  Thanks!

File: tests/integration/test_script_runner.py [hunk 2]
```diff
@@ -364,6 +364,12 @@
         user_flow_script="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_sqlite_data.py",
         data_context_dir="tests/integration/fixtures/no_datasources/great_expectations",
     ),
+    IntegrationTestFixture(
```
Comment (anthonyburdi) [inline tests/integration/test_script_runner.py:360]: Recently we split up test definitions by backend so we could parallelize / isolate in `tests/integration/test_definitions/`, could you move this to `tests/integration/test_definitions/postgresql/integration_tests.py`?

Comment (alexsherstinsky) [reply tests/integration/test_script_runner.py:360]: @TrangPham  @anthonyburdi and I agreed that I will refactor this once my PRs merge -- and then everybody can use it.  If you want to merge yours before the refactor, please let me know (may be fewer merge conflicts for you to resolve this way).  So if you can address the questions regarding this PR soon, I am happy to re-review right away.  Thanks!

Comment (TrangPham) [reply tests/integration/test_script_runner.py:360]: That works.Could you approve this PR? @alexsherstinsky

**Review by alexsherstinsky — APPROVED** [review]
LGTM

**Review by alexsherstinsky — CHANGES_REQUESTED** [review]
@TrangPham I am so sorry -- I feel that I want to bring up a couple of questions for consideration:
1) We probably want to test with SQLite for this section of the documentation (i.e., this is not a comprehensive backends test).  What do @anthonyburdi and @Shinnnyshinshin think?
2) We need to add the loading of a batch and asserting that we were able to get the data.

Thank you.

**Comment by TrangPham** [comment]
@alexsherstinsky I updated to make assertions on getting the data and also changing to use sqlite instead of postgres

**Review by alexsherstinsky — APPROVED** [review]
LGTM!

**Pull request merged**